go-controller/pkg/node/gateway_iptables.go

```diff
@@ -587,3 +559,17 @@ func egressSVCIPTRulesForEndpoints(svc *kapi.Service, v4Eps, v6Eps []string) []i
 
 	return rules
 }
+
+func egressSVCIPTDefaultReturnRule() []iptRule {
```

Comment:
Only the first element of the returned array is used. Can you turn the return type to `iptRule`?

Reply:
done.